designs/virtualization/4.x/Templates/Templates.md

```diff
@@ -0,0 +1,57 @@
+# Templates
+
+## Virtual machines and VM templates are located under Virtualization
+
+Removing the tab ‘Virtual Machine Templates’ from the side navigation and combining it with ‘Virtual Machines’, so both are located under ‘Virtualization’.
+This PR will focus on how users can create a VM when they choose a to create a VM from a Template.
+
+### Virtual Machine Templates Tab
+
+From the Virtualization left nav item the user goes to the Virtual Machine Templates Tab where they can see the full list of templates.
+From this point, we suggest 2 options to create a VM:
+
+- Create VM from the selected templated via an action from the kebab menu
+
+![TemplatesTabOp1](img/TemplatesTabsOp1.png)
+
+- Create VM from the selected templated via an action from the kebab menu and also allow an action link
+
+![TemplatesTabOp2](img/TemplatesTabsOp2.png)
+
+Both options will open a modal where the user will need to provide a name for the VM they want to create.
+Users can click the button link ‘View Template details’ to view the Template details page.
```

Comment:
Would this action take them out of the flow, I wonder if we should have it as an external link so they don't exit the modal. What do you think?

Reply:
+1